src/lib/chromedriver.js

```diff
@@ -1,4 +1,4 @@
-var chromedriver = require('chromedriver'),
+var chromedriver = module.parent.require('chromedriver'),
```

Comment:
I think this means we break tests in the chimpy itself. Maybe we should have the chromedriver as both peerDependency and devDependnecy, so when installing in other projects it doesn't come with it's own, but when installed on its own it does have a chromedriver to use